tex/courses/subjects/INFS4205.tex

```diff
@@ -9,19 +9,19 @@
 Assignment 2 & 25\% & Group Assignment \\
-Final Exam & 55\% & Open book short answer exam \\
+Final Exam & 50\% & Open book short answer exam \\
 },
 review = {
-    This course provides an overview on the of the theory of high dimensional databases, with a focus on indexing and searching. This content builds on the theory taught in INFS2200 and it is highly recommended to have taken that course in the past. In short, this course continues to build on the indexing and query planning parts of INFS2200.\\
+    This course provides an overview of the theory of high dimensional databases, with a focus on indexing and searching. This content builds on the theory taught in INFS2200 and it is highly recommended to have taken that course in the past. In short, this course continues to build on the indexing and query planning components of INFS2200.\\
 
     The only contact hours for the subject are weekly 2 hour lectures. While attendance is not required, class discussion usually goes on in them, meaning that turning up can be quite beneficial.\\
 
-    There are 2 assignments with the first being a relatively easy theory assignment and the second being a group assignment. The first assignment is just several theory questions that you must answer, where the second assignment involves choosing a project idea relating to the course content and working on that. Examples of this are data driven software projects powered by a spatial database, to in-depth theoretical analysis of algorithms that were taught in the course. The final submission for this assignment is a presentation and research report.\\
+    There are 2 assignments with the first being a relatively easy theory assignment and the second being a group assignment. The first assignment is just several theory questions that you must answer, where the second assignment involves choosing a project idea relating to the course content and working on that. Examples of this are data driven software projects powered by a spatial database, or an in-depth theoretical analysis of algorithms that were taught in the course. The final submission for this assignment is a presentation and research report.\\
 
-    The final exam is open book and can be quite difficult. Even though the theory behind it is fairly simple as long as you are familiar with the course content, the questions require in depth answers that can take a large amount of time to answer. This means that the main challenge in the exam is making small mistakes on rushed answers, rather then trouble with the overall theory.\\
+    The final exam is open book and can be quite difficult. Even though the theory behind it is fairly simple, as long as you are familiar with the course content, the questions require in depth answers that can take a large amount of time to answer. This means that the main challenge in the exam is making small mistakes on rushed answers, rather then trouble with the overall theory.\\
```

Comment:
" as long as you are familiar with the course content" should either be in brackets IMO or completely removed.